sentences/ro/_common.yaml

```
@@ -1,12 +1,12 @@
 language: ro
 responses:
   errors:
-    no_intent: "Scuze, nu am înțeles cererea"
-    no_area: "Nu există nici o arie numită {{ area }}"
-    no_domain: "Aria {{ area }} nu conține {{ domain }}"
-    no_device_class: "Aria {{ area }} nu conține {{ device_class }}"
-    no_entity: "Niciun dispozitiv sau entitate cu numele {{ entity }}"
-    handle_error: "A apărut o eroare în timpul procesării cererii"
+    no_intent: "Îmi pare rău, nu am înțeles. Poți, te rog, să repeți?"
+    no_area: "Îmi pare rău, nu este nici o cameră numită {{ area }}"
+    no_domain: "Îmi pare rău, în {{ area }} nu este {{ domain }}"
+    no_device_class: "Îmi pare rău, în {{ area }} nu este {{ device_class }}"
+    no_entity: "Îmi pare rău, nu am găsit nici un dispozitiv sau entitate cu numele {{ entity }}"
```

Forma corecta e "niciun" https://dexonline.ro/article/Niciun_sau_nici_un%3F
```suggestion
    no_entity: "Îmi pare rău, nu am găsit niciun dispozitiv sau entitate cu numele {{ entity }}"
```